charts/pomerium/templates/signing-key-secret.yaml

```diff
@@ -0,0 +1,31 @@
+{{- define "pomerium.proxy.signingKeySecretObject" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ template "pomerium.name" . }}
+    helm.sh/chart: {{ template "pomerium.chart" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+  annotations:
+    helm.sh/hook-delete-policy: before-hook-creation
+{{- if .Values.config.forceGenerateSigningKey }}
+    helm.sh/hook: pre-upgrade
+{{- else if .Values.config.generateSigningKey }}
+    helm.sh/hook: pre-install
+{{- end }}
+  name: {{ template "pomerium.proxy.signingKeySecret.name" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
@tgomas helm already takes care of setting the namespace, it's bad practice to include it in templates

Reply:
@victornoel both basic and tls secrets templates have same namespace definition, I wonder why ?

Also do you recommend to remove namespace definition in this PR from all secrets templates, or only from signing-key-secret.yaml and optionally from other secrets in a separate PR ?

Reply:
@tgomas it's a mistake IMHO, let's remove all of them if @travisgroth is ok, if not just for this one then :)

Reply:
Removed for all secrets template, can be changed if @travisgroth has another point of view

Reply:
Likely just been copied and pasted.  👍 on removing them.